guides/release/components/template-tag-format.md

```diff
@@ -259,6 +259,18 @@ export default CustomSelect;
 
 This can be a powerful refactoring technique to break up large components into smaller ones. (where it makes sense!)
 
+## Runtime compiler
```

Comment:
I think this section is missing some explanation for context?

Reply:
aye, we need to copy a bunch of stuff from here: https://github.com/emberjs/rfcs/pull/931